### PR TITLE
Add steps to change db2 license file

### PR DESCRIPTION
**Pull request by Doout**
Add in steps to apply the prod `lic` file for db2.

### CODE REVIEW

File: deployment/README.md [hunk 1]
```diff
@@ -257,7 +257,18 @@ Max amount of table space (GB):   "100"
 ```
 
 ### 3. Apply the production license
-Add steps of configmap update with db2adv_vpc.lic
+
+Update `RELEASE_NAME` to the namespace/project where db2 is deploy and `LICENSE_FILE` to the location of the `lic` file
+
+```
+RELEASE_NAME="db2u-cn1"
```
Comment (rayyaoibm) [inline deployment/README.md]: I will suggest we use the consistent example. Earlier in the document, we used "db2u-release-1" as the value of RELEASE_NAME, can use the same value here?

File: deployment/README.md [hunk 2]
```diff
@@ -257,7 +257,18 @@ Max amount of table space (GB):   "100"
...
+LICENSE_FILE="db2adv_vpc.lic"
```
Comment (rayyaoibm) [inline deployment/README.md]: Can we add in the path to emphasize path is needed? 
LICENSE_FILE="./db2adv_vpc.lic"

**Review by rayyaoibm — COMMENTED** [review]
Thank you for the update. Can you address the minor comments?

File: deployment/README.md (hunk 1 above)
Comment (Doout) [reply deployment/README.md]: we can

File: deployment/README.md (hunk 2 above)
Comment (Doout) [reply deployment/README.md]: I will add that.

**Review by rayyaoibm — APPROVED** [review]

**Pull request merged**